src/index.html

```diff
@@ -1,12 +1,346 @@
+          </li>
+          <li class="nav__item">
+            <a href="#phones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#computers" 
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-content">
+      <div class="product-card">
+        <div class="product-card__wrap" data-qa="card">
+          <img
+            class="product-card__image"
+            src="./images/imac.jpeg"
+            alt="Моноблок APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+          >
+          <h2 class="product-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">
+            Код товара: 195434
+          </p>
+          <div class="product-card__rating">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="product-card__reviews">
+              Отзывов: 5
+            </div>
+          </div>
+          <div class="product-card__price-box">
+            <div class="product-card__price-name">Цена:</div>
+            <div class="product-card__price-value">69 999 грн</div>
+          </div>
+          <a
+            href="#"
+            class="product-card__button"
+            data-qa="card-hover"
+          >
+            купить
+          </a>
+        </div>
+        
+        <div class="product-card__wrap">
+          <img
+            class="product-card__image"
+            src="./images/imac.jpeg"
+            alt="Моноблок APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+          >
+          <h2 class="product-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">
+            Код товара: 195434
+          </p>
+          <div class="product-card__rating">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
```

Comment:
додай до класу модифікатори щоб при зміні модифікатора змінювалась кількість активних зірок
```suggestion
          <div class="product-card__rating">
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```